.github/pull_request_template.md

```diff
@@ -19,6 +19,25 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 
 - [ ] Test A
 
+# Does This PR Require a Contrib Repo Change?
+
+Answer the following question based on these examples of changes that would require a Contrib Repo Change:
+- [The OTel specification](https://github.com/open-telemetry/opentelemetry-specification) has changed which prompted this PR to update the method interfaces of `opentelemetry-api/` or `opentelemetry-sdk/`
+- The method interfaces of `opentelemetry-instrumentation/` have changed
+- The method interfaces of `test/util` have changed
+- Scripts in `scripts/` that were copied over to the Contrib repo have changed
```

Comment:
just out of curiosity: since there's a dependency on the cloned core repo, couldn't the contrib files use the cloned scripts instead?

Reply:
I think there are subtle differences that apply for only a few scripts.

For example `package.sh` in Contrib would need to go through a `instrumentation/` library but Core (soon) won't have to.

There used to be a discrepancy between the `eachdist.py` scripts but slowly they ended up doing the same thing. However if we want to make cloning the "other" repo as a script command in the future that would be another point where they could deviate.

I thought to add this point because there are cases where a feature enhancement could benefit both repos, and cases where a repo specific feature wouldn't benefit the other  ¯\_(ツ)_/¯